src/Utils/BlockTemplateUtils.php

```diff
@@ -3,10 +3,9 @@
 
 use Automattic\WooCommerce\Blocks\Domain\Services\FeatureGating;
 
-
 /**
- * BlockTemplateUtils class used for serving block templates from Woo Blocks.
- * IMPORTANT: These methods have been duplicated from Gutenberg/lib/full-site-editing/block-templates.php as those functions are not for public usage.
+ * Utility methods used for serving block templates from WooCommerce Blocks.
+ * {@internal This class and its methods should only be used within the BlockTemplateController.php and is not intended for public use. }}
```

Comment:
There are two closing `}}` instead of one. I suppose this is a mistake. I think in general it looks ok without brackets and it's still valid: https://docs.phpdoc.org/guide/references/phpdoc/tags/internal.html

Reply:
It was intentional, I've seen both used in PHP and its documentation. Perhaps there is a reason I'm not aware of for double closing `}}` rather than single. Either way, happy to remove one of the two!

Reply:
Ok so I found it, in [this old reference](https://docs.phpdoc.org/2.9/references/phpdoc/tags/internal.html):

> The inline version of this tag may, contrary to other inline tags, contain text but also other inline tags. To increase readability and ease parsing should the tag be terminated with a double closing brace, instead of a single one.

Note that this is only for the inline version of the tag. Also note that in the most updated version of the docs ([linked above](https://docs.phpdoc.org/guide/references/phpdoc/tags/internal.html)), basically the same page, this sentence is not present.

It is also quite ironic for me that the reasoning given was to “increase readability”. While searching for this, I found this funny thread on `r/lolphp`: https://www.reddit.com/r/lolphp/comments/4i27ud/to_increase_readability_should_the_tag_be/

I would perhaps consider removing the braces altogether. It's valid syntax both in the old and new version. What do you think?

Reply:
I removed one of the two braces yesterday, like you suggested. Looking at other usages in the WC codebase that looks more consistent with what is there. Ill merge for now, and we can always remove them in another PR if we feel the need to